docs/pages/versions/unversioned/sdk/appearance.md

```diff
@@ -2,27 +2,36 @@
 title: Appearance
 ---
 
-Detect preferred color scheme (light, dark, or no preference) on iOS 13+.
+Detect preferred color scheme (`light`, `dark`, `automatic` or `no-preference`) on iOS 13+ and Android 10+.
```

Comment:
```suggestion
Detect preferred color scheme (`light`, `dark`, or `no-preference`) on iOS 13+ and Android 10+.
```

`automatic` won't be returned by `Appearance.getColorScheme()` only [light, dark, and no-preference](https://github.com/expo/react-native-appearance/blob/4022f579b07469b1e0ff4fabf113827a7a4c43d2/src/Appearance.types.ts#L1)